src/cli/options.rs

```diff
@@ -2352,21 +2344,21 @@ pub enum CaCommand {
     Delete(Handle),     // Delete the CA -> let it withdraw and request revocation as well
 
     // Publishing
-    RepoPublisherRequest(Handle), // Get the RFC8183 publisher request
+    RepoPublisherRequest(Handle), // Get the idexchange:: publisher request
```

Comment:
You probably didn't mean to change this ;-)

Reply:
fixed